apps/frontend/src/index.css

```
@@ -80,16 +92,17 @@
     display:inline-flex; align-items:center; gap:6px;
     font-family:var(--mono-display); font-weight:800; font-size:11px;
     letter-spacing:0.14em; text-transform:uppercase;
-    border:2px solid currentColor; border-radius:3px;
+    border:2px solid currentColor; border-radius:var(--radius-md);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)